packages/web-lib/src/persistence/strategies/local-storage.strategy.ts

```diff
@@ -0,0 +1,69 @@
+import { IStorageStrategy, OperationResult } from '../web-store.types';
+
+export class LocalStorageStrategy implements IStorageStrategy {
+  storage: Storage;
+
+  constructor(storage?: Storage) {
+    this.storage = storage ?? localStorage;
+  }
+
+  async init(): Promise<void> {}
+
+  async getOneByKey<TData>(key: string): Promise<OperationResult<TData>> {
+    try {
+      const storedItem = this.storage.getItem(key);
+      if (!storedItem) {
+        return { success: false, data: 'nothing' as TData };
+      }
+      return {
+        success: true,
+        data: JSON.parse(storedItem) as TData,
+      };
+    } catch (error) {
+      const errorMsg = 'Error in local storage getOneByKey: ';
+      if (error instanceof SyntaxError) {
+        console.error(errorMsg + error.name);
+      } else if (error instanceof Error) {
+        console.error(errorMsg + error.message);
+      }
+      return { success: false, data: 'error' as TData };
```

Comment:
we need to create AvxError here and return it in error. Looks like I missed pattern used here for result. We will review it in 1x1 and probably re-implement